src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -541,30 +548,30 @@ private static RefKind GetModifiers(SyntaxTokenList modifiers, out SyntaxToken r
                 switch (modifier.Kind())
                 {
                     case SyntaxKind.OutKeyword:
-                        refOrOutKeyword = modifier;
+                        refnessKeyword = modifier;
                         if (refKind == RefKind.None)
                         {
                             refKind = RefKind.Out;
                         }
                         break;
                     case SyntaxKind.RefKeyword:
-                        refOrOutKeyword = modifier;
+                        refnessKeyword = modifier;
                         if (refKind == RefKind.None)
                         {
                             refKind = RefKind.Ref;
                         }
                         break;
                     case SyntaxKind.InKeyword:
+                        refnessKeyword = modifier;
```

Comment:
Good point. I guess it should be fixed as well.

---
In reply to: [115866313](https://github.com/dotnet/roslyn/pull/18715#discussion_r115866313) [](ancestors = 115866313)